### DISCUSSION TITLE
Slovak translation

### DISCUSSION THREAD

**Discussion by m-cerveny** [ChristopherKlay/StadiaEnhanced#142] [Translations]
var translation = {
    default: 'Pôvodné',
    native: 'Natívne',
    hide: 'Skry',
    show: 'Zobraz',
    total: 'Celkovo',
    visible: 'Viditeľné',
    hidden: 'Skryté',
    enabled: 'Povolené',
    disabled: 'Zakázané',
    auto: 'Automatické',
    manual: 'Manuálne',
    all: 'Všetky',
    locked: 'Uzamknuté',
    complete: 'Kompletné',
    incomplete: 'Nekompletné',
    games: 'Hry',
    allgames: 'Všetky hry',
    ***leavepro: 'Hry, ktoré čoskoro opustia Stadia Pro',
    ***bundles: 'Balíčky',
    addons: 'Doplnky',
    wishlist: 'Wishlist',
    responsive: 'Dynamické',
    windowed: 'V okne',
    fullscreen: 'Na celú obrazovku',
    ***progames: 'Hry Stadia Pro',
    ***onsale: 'Výpredaj',
    ***prodeals: 'Výhodné ponuky Stadia Pro',
    trials: 'Skúšobné verzie',
    userprofile: 'Môj profil',
    usermedia: 'Zábery a herné situácie',
    searchbtnbase: 'Vyhľadaj',
    avatarpopup: 'URL ku novému avataru  (ponechaj prázdne pre základný):',
    date: 'Dátum',
    time: 'Čas',
    sessiontime: 'Uplynutý čas',
    codec: 'Kodek',
    resolution: 'Rozlíšenie',
    hardware: 'Hardware',
    software: 'Software',
    trafficsession: 'Celkový dátový prenos',
    trafficcurrent: 'Aktuálny dátový prenos',
    trafficaverage: 'Priemerný dátový prenos',
    packetloss: 'Stratené pakety',
    framedrop: 'Stratené frejmy',
    latency: 'Odozva',
    jitter: 'Jitter buffer',
    decodetime: 'Trvanie dekódovania',
    compression: 'Kompresia',
    detailed: 'Detailné',
    compact: 'Skrátené',
    streammon: 'Monitoring dátového prenosu',
    stream: 'Dátový prenos',
    network: 'Sieť',
    session: 'Relácia',
    extdetail: 'Rozšírené detaily',
    maxresolution: 'Maximálne rozlíšenie',
    fps: 'Framerate',
    testdiscl: '<b>Upozornenie:</b> Táto hra zatiaľ nebola testovaná.',
    datadiscl: 'Toto je maximálny framerate počas hrania v 4K móde (vyžaduje Pro odber).\
    Pri hrách ktoré majú rozlíšenie/framerate možnosť, je uprednostnené rozlíšenie.\
    Tieto údaje sú poskytnuté od <a href="https://twitter.com/OriginaIPenguin" target="_blank">@OriginaIPenguin</a> \
    a kompletná databáza sa nachádza <a href="https://linktr.ee/StadiaDatabase" target="_blank">tu</a>.',
    noteOne: '4K Mód',
    noteTwo: '30/60 FPS možnosť',
    noteThree: '60 FPS pri 1080p móde',
    noteFour: '30 FPS pri 1080p móde',
    noteFive: 'Neumožňuje 4K mód',
    unsupported: 'Nepodporované',
    crossfriends: 'Cross-platform systém kamošov nieje podporovaný',
    filtersettings: 'Nastavenia filtra',
    saturation: 'Sýtosť',
    contrast: 'kontrast',
    brightness: 'Jas',
    sharpen: 'Ostrosť',
    community: 'Komunita',
    speedtest: 'Test rýchlosti',
    quickaccess: 'Rýchly prístup',
    messages: 'Správy',
    comfeature: 'Komunitné funkcie',
    avatar: 'Avatar',
    interface: 'Rozhranie',
    shortcut: 'StadiaIcons',
    shortcuttitle: 'Vytvor skratku ku hre',
    shortcutdesc: 'Umožňuje inštaláciu skratky do hry na tomto zariadení.',
    stadiadatabase: 'Stadia Database',
    stadiadatabasedesc: 'Zobrazí sekciu "Rozšírené detaily" v Obchode, napríklad framerate, rozlíšenie a viac informácií o hre.',
    contactdev: 'Kontaktuj vývojára/distribútora',
    stadiahunters: 'Stadia Hunters',
    stadiahuntersdesc: 'Prístup ku Stadia Hunters komunite, vrátane sledovania ocenení, návodov, rebríčkov a iných. Výborný pomocník pre lovca ocenení.',
    stadiahunterstitle: 'na Stadia Hunters',
    stadiahunterslogin: 'Klikni a prihlás sa',
    stadiahuntersnotfound: 'Používateľ neexistuje',
    stadiahunterslevel: 'ˇUroveň',
    stadiahuntersworldrank: 'Pozícia vo svete',
    stadiahuntersxphover: 'Pokrok úrovne',
    gridsize: 'Veľkosť mriežky herných ikon',
    griddesc: 'Nastavuje počet herných ikon na riadok v knižnici hier.',
    clock: 'Hodiny',
    clockdesc: 'Zobrazuje aktuálny čas v zozname priateľov, v hernom info panely, alebo v oboch.',
    friendslist: 'Zoznam priateľov',
    igoverlay: 'Herný info panel',
    listoverlay: 'Priatelia a info panel',
    filter: 'Filter hier',
    filterdesc: 'Filter umožňuje sprehľadnenie knižnice tým že skryje nechcené hry pomocou prepínača. Prepínač sa nachádza v pravo hore nad knižnicou hier.',
    invitebase: 'Skopíruj pozývací link',
    inviteactive: 'Skopírované!',
    gamelabel: 'Visačky',
    gamelabeldesc: 'Odstráni "Pro" a iné visačky z hier na domovskej obrazovke.',
    dimoverlay: 'Stmavené pozadie',
    dimoverlaydesc: 'Odstráni stmavené pozadie pod otvoreným Stadia menu počas hrania.',
    homegallery: 'Používateľova galéria',
    homegallerydesc: 'Skryje sekciu "Zábery a herné situácie" na spodku domovskej obrazovky.',
    quickprev: 'Náhľad správ',
    quickprevdesc: 'Skryje náhľady správ v zozname priateľov.',
    quickrep: 'Rýchla odpoveď',
    quickrepdesc: 'Skryje možnosť rýchlej odpovede v čete.',
    offlinefriend: 'Offline priatelia',
    offlinefrienddesc: 'Skryje offline priateľov v zozname.',
    invisiblefriend: '"Neviditeľní" priatelia',
    invisiblefrienddesc: 'Skryje priateľov s "neznámym" statusom v zozname.',
    notification: 'Notifikácie',
    notificationdesc: 'Zobrazenie notifikácií keď je Stadia Enhanced aktualizovaná na novú verziu. ("Automatické" skrytie notifikácie po 5 sekundách, "Manuálne" skrytie notifikácie po prvej používateľovej interakcii).',
    streammode: 'Súkromie pri zdielanom prenose',
    streammodedesc: 'Zahmlí niektoré elementy používateľovho rozhrania (napr. Zoznam priateľov, atď.) počas zdielaného prenosu (streamovania) na iné platformy a pri zdielaní cez streamovacie nástroje (napr. OBS / Discord).',
    forcefocus: 'Udrž aktívnosť',
    forcefocusdesc: 'Umožní hrám byť aktívne aj na pozadí.',
    catprev: 'Visačky kategórií',
    catprevdesc: 'Skryje visačky herných kategórií v info náhľade hry pod kurzorom myši.',
    streammondesc: 'Aktivuj Monitoring dátového prenosu ihneď ako sa spustí hra.',
    resolutiondesc: 'Požadované rozlíšenie. 1440p a 2160p módy vyžadujú VP9 kodek.',
    codecdesc: 'Kodek použitý počas hrania.',
    confirmreset: 'Si si istý že chceš vymazať nastavenia?',
    statistics: 'Štatistiky',
    gamesfinished: 'Prejdené hry',
    achievements: 'Ocenenia',
    achievementsunlocked: 'Dosiahnuté ocenenia',
    totalPlayTime: 'Celkový čas hrania',
    splitstore: 'Viacej hier na stranu',
    splitstoredesc: 'Zobrazí hry v obchode do dvoch stĺpcov. Prehľadnejšie, menej skrolovania.',
    inlineimage: 'Náhľad obrázku',
    inlinedesc: 'Nahradí linky ku obrázkom v štandardných formátoch za kliknuteľné náhľady.',
    familyelements: 'Zdieľanie v rámci rodiny',
    familyelementsdesc: 'Skryje "Zdieľať túto hru s rodinou" možnosť v info náhľade.',
    ***donations: 'Prispieť',
    reportbug: 'Nahlásiť chybu',
    exportset: 'Export nastavení',
    importset: 'Import nastavení',
    ***importerror: 'Súbor ktorý sa snažíte otvoriť neobsahuje platný Stadia Enhanced profil.',
    resetsettings: 'Vymazať nastavenia'
}

**Comment by m-cerveny** [comment]
@ChristopherKlay Slovak translation is production ready. Just FYI...
Fantastic job, BTW!

DIFF tool: https://editor.mergely.com/

**Reply by ChristopherKlay** [reply]
Will go live with `1.15.1`.

Thank you for the work!

**Reply by m-cerveny** [reply]
Thank you! Lightning fast!!

**Comment by m-cerveny** [comment]
@ChristopherKlay I have synced all translations with official Stadia wording. Now it will feel much more natural to the users.
You may include this in the next version (whenever you have the time).

Thank you, SE is a fantastic extension!!

**Comment by m-cerveny** [comment]
@ChristopherKlay it seems you forgot the update of Slovak translation in 1.15.2. It was done 5 days ago.
Never mind, in the next version... ;)

**Reply by ChristopherKlay** [reply]
My bad - i thought it was already included in the dev build by then.
Will hotfix it in during the next 24 hours (:

**Reply by m-cerveny** [reply]
No worries. Thanks for the effort!

**Comment by m-cerveny** [comment]
Hello @ChristopherKlay!
Slovak translation updated with import/export texts. Ready to be published. Thank you!

**Comment by m-cerveny** [comment]
Hello @ChristopherKlay, new Slovak translation updated in the first post. Thank you for your fantastic work!!

**Comment by m-cerveny** [comment]
@ChristopherKlay you forgot to include new Slovak translation in 1.20.7. ;)

**Reply by ChristopherKlay** [reply]
Next time, please leave a comment with the new strings — that way around i actually get a notification and other people can see what changed between the versions, without checking the difference between every history state of the initial comment.

It'll be patched live with `1.20.8` in a moment (:

**Reply by m-cerveny** [reply]
Thanks for the hint. It was a huge update - I skipped quite a few versions. I'll do as you suggest.
Great work!
